docs/tutorials/Forecast Uber Stock Prices using MindsDB.mdx

```diff
@@ -0,0 +1,89 @@
+![image](https://user-images.githubusercontent.com/26898623/236202668-20f5656d-6c32-4c4a-a00e-2517e36ba386.png)
+
+3. Click on Add Data from the highest right and on consequent screen that seems, select Databases from left panel and so click on Mysql
+![image](https://user-images.githubusercontent.com/26898623/236202789-e42f6b3f-a54a-46cc-9b07-92bd1a9533bc.png)
+
+4. we need to provide certain parameters to establish a connection to our local database. In order to that, we need to expose localhost:3306 to the internet so that we can establish a tcp/ip connection to interact with it
+![image](https://user-images.githubusercontent.com/26898623/236202883-2121e272-1273-441b-bf6c-c47db02fcf20.png)
+
+5. We have to now use ngrok to tunnel our localhost to the internet. This command should do the job for you.  
+`ngrok tcp 3306`
+
+![image](https://user-images.githubusercontent.com/26898623/236204171-05310422-eddc-44ec-8f77-a99097d36095.png)
+
+6. We can now fill in the parameters to connect to the database and then hit the test connection button.
+![image](https://user-images.githubusercontent.com/26898623/236204351-f8026d67-6320-4e8e-a4a7-154468c9ea38.png)
+
+7. Now let's run a simple SELECT query now to make sure that MindsDB can now access the data in the database.   
+` select * from MindsDBUber.uber`
+![image](https://user-images.githubusercontent.com/26898623/236204572-a7762077-d8b8-43fb-b8aa-2beef92eab5e.png)
+
+# Creating a Predictor Model
+To create a Predictor model with MindsDB, all you need to do is execute a simple SQL query. Let's explore how straightforward it is to create this model.
+
+1. To create and train a new model using our available dataset, we can use MindsDB's CREATE PREDICTOR function. The query is provided below for your convenience.   
+
+ ![image](https://user-images.githubusercontent.com/26898623/236204921-af3a0dd7-f7cd-463e-bc71-892f24403b8a.png)
+
+2. The model can take some time to be ready. Meanwhile, we can check its status using the command below.  
+` SELECT status
+ FROM mindsdb.predictors
+ WHERE name = 'uber_predictor';`
+ ![image](https://user-images.githubusercontent.com/26898623/236205284-3cbe21e1-c140-4cea-90a1-dfa20b409a92.png)
+
+# Forecasting Target Values
+As usual, MindsDB also offers simple SQL statements to predict or forecast values from the model. Let's explore how we can forecast values.
+
+Before we proceed, let's consider the basic scenario here. We have a dataset that lists the closing stock values of Uber over a period of time. Using this predictor model, we will attempt to predict the closing value after the last record available in the dataset. We can achieve this by using the keyword LATEST for the current dates.
+
+The query will be a straightforward SELECT statement, where we will join the predictor model with the data table and retrieve the forecasts.
+
+`
+SELECT T.Date as date,
```

Comment:
For all the SQL queries/codes, please use a code block, as below:

\```sql
SQL GOES HERE
\```

Reply:
done